socketioxide/src/adapter.rs

```diff
@@ -173,9 +173,14 @@ impl Adapter for LocalAdapter {
         let sockets = self.apply_opts(opts);
 
         tracing::debug!("broadcasting packet to {} sockets", sockets.len());
-        sockets
-            .into_iter()
-            .try_for_each(|socket| socket.send(packet.clone()))
+        sockets.into_iter().try_for_each(|socket| {
+            if let Err(SendError::SocketFull { .. }) = socket.send(packet.clone()) {
+                // todo skip message? return err? try send later? create delayed queue?
```

Comment:
I would say, skip message and return a vec of errors ?
Or create a BroadcastError that can take a list of sids and the rejected packet.